syft/pkg/cataloger/javascript/parse_pnpm_lock.go

```diff
@@ -36,6 +38,19 @@ func parsePnpmLock(resolver source.FileResolver, _ *generic.Environment, reader
 		pkgs = append(pkgs, newPnpmPackage(resolver, reader.Location, name, version))
 	}
 
+	// parse packages from packages section of pnpm-lock.yaml
+	for nameVersion := range lockFile.Packages {
+		nameVersionSplit := strings.Split(nameVersion, "/")
```

Comment:
maybe just `strings.TrimPrefix(nameVersion, "/")` and don't ignore the first element later? are there situations where there is something else at the beginning of these strings?

Reply:
Oh yeah doing like this

```go
nameVersionSplit := strings.Split(strings.TrimPrefix(nameVersion, "/"), "/")
```

would probably be easier. I believe in the `packages:` section of the `pnpm-lock.yaml` every package will start with `/`.

Along with that, should I also do something like

```go
// last element in split array is version
version := nameVersionSplit[len(nameVersionSplit)-1]
nameVersionSplit = nameVersionSplit[:len(nameVersionSplit)-1] // remove version

// construct name from all array items
name := strings.Join(nameVersionSplit, "/")
```

instead of 

```go
// last element in split array is version
version := nameVersionSplit[len(nameVersionSplit)-1]

// construct name from all array items other than last item (version)
name := strings.Join(nameVersionSplit[:len(nameVersionSplit)-1], "/")
```

or which would be preferred?